lib/src/model/discovered_device.dart

```diff
@@ -27,7 +27,14 @@ class DiscoveredDevice extends $DiscoveredDevice {
   @CustomEquality(DeepCollectionEquality())
   final Map<Uuid, Uint8List> serviceData;
 
-  const DiscoveredDevice({@required this.id, @required this.name, @required this.serviceData});
+  @CustomEquality(Ignore())
```

Comment:
Why is this field ignored for equality?

Reply:
My reasoning behind it is that a device with the same id, service data and name but with a different RSSI would be seen as a different object and I think this shouldn't be the case.